pkgs/development/python-modules/adb-enhanced/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, buildPythonPackage, docopt, psutil, pythonOlder }:
+
+buildPythonPackage rec {
+
+  pname = "adb-enhanced";
+  version = "2.5.9";
+
+  disabled = pythonOlder "3.4";
+
+  src = fetchFromGitHub {
+    owner = "ashishb";
+    repo = "adb-enhanced";
+    rev = version;
+    sha256 = "08fmb55p80fbx2cix6qv2gpw3fi5ic3q17vzvza9brfwx6z1g6dv";
+  };
+
+  propagatedBuildInputs = [ psutil docopt ];
+
+  # Disable tests because they require a dedicated android emulator
+  doCheck = false;
+
+  pythonImportsCheck = [ "psutil" "docopt" ];
+  meta = with lib; {
+    description = "Wrapper around adb to trigger various scenarios like screen rotation, battery saver mode, data saver mode, doze mode, permission grant/revocation";
```

Keep it simple. Also, using `adb wrapper` suggests that `adb` from `android-tools` is used under hood which is not the case.

```suggestion
    description = "Tool for Android testing and development";
```

Use `longDescription` for multi-line and additional information.